.gitignore

```diff
@@ -1 +1,36 @@
-.vscode/
```

Comment:
Maybe need to add editor config .gitignore back in if others are using VSCode too

Reply:
Great point